src/report.sh

```diff
@@ -386,9 +411,21 @@ function parse_report_options()
         shift 2
         ;;
       --output | -o)
+        if [[ "${options_values['STATISTICS']}" == 1 ]]; then
+          options_values['ERROR']='--output cannot be used with --statistics.'
+          return 22 # EINVAL
+        fi
```

Comment:
Check this condition after parsing all arguments, as to not impose an order on the options.